e2e/cypress/integration/api/iam/04_projects_api_client_runs_ingestion.ts

```diff
@@ -0,0 +1,291 @@
+    projectsWithRule.forEach(projectWithRule => {
+      cy.request({
+        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+        method: 'POST',
+        url: '/apis/iam/v2beta/projects',
+        body: projectWithRule.project
+      });
+
+      cy.request({
+        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+        method: 'POST',
+        url: `/apis/iam/v2beta/projects/${projectWithRule.rule.project_id}/rules`,
+        body: projectWithRule.rule
+      });
+    });
+
+    // apply rules
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'POST',
+      url: '/apis/iam/v2beta/apply-rules'
+    });
+
+    waitUntilApplyRulesNotRunning(100);
+  });
+
+  after(() => cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']));
+
+  it('tagging 7 projects with all the attributes on a client runs node', () => {
+    const nodeId = uuidv4();
+
+    // Ensure there are no nodes matching any of the projects
+    projectsWithRule.forEach(projectWithRule => {
+      cy.request({
+        headers: {
+          'api-token': Cypress.env('ADMIN_TOKEN'),
+          projects: projectWithRule.project.id
+        },
+        method: 'GET',
+        url: '/api/v0/cfgmgmt/nodes?pagination.size=10'
+      }).then((response) => {
+        expect(response.body).to.have.length(0);
+      });
+    });
+
+    // Ingest a node with attribues that match all the projects
+    cy.fixture('converge/avengers1.json').then((node) => {
+      node.organization_name = '75th Rangers';
+      node.chef_server_fqdn = 'example.org';
+      node.node.chef_environment = 'arctic';
+      node.policy_group = 'red_ring';
+      node.policy_name = 'fire';
+      node.node.automatic.roles = ['backend'];
+      node.node.normal.tags = ['v3'];
+      node.entity_uuid = nodeId;
+      cy.request({
+        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+        method: 'POST',
+        url: '/data-collector/v0',
+        body: node
+      });
+    });
+
+    // wait for the node to be ingested
+    waitForNodes(projectsWithRule[0].project.id, 30);
+
+    // Ensure the node is tagged with the correct project
+    projectsWithRule.forEach(projectWithRule => {
+      cy.request({
+        headers: {
+          'api-token': Cypress.env('ADMIN_TOKEN'),
+          projects: projectWithRule.project.id
+        },
+        method: 'GET',
+        url: '/api/v0/cfgmgmt/nodes?pagination.size=10'
+      }).then((response) => {
+        expect(response.body).to.have.length(1);
+        expect(response.body[0].id).to.equal(nodeId);
+      });
+    });
+
+    // Delete node
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'POST',
+      url: 'api/v0/ingest/events/chef/node-multiple-deletes',
+      body: {
+        node_ids: [nodeId]
+      },
+      failOnStatusCode: false
+    });
+  });
+});
+
+function waitUntilApplyRulesNotRunning(attempts: number): void {
```

Comment:
now that this is in two different files, I would make it a commands helper. There's a little setup required to do that.
1. add the function as a Cypress command to [commands.ts](https://github.com/chef/automate/blob/master/e2e/cypress/support/commands.ts)
2. add the function signature to [index.d.ts](https://github.com/chef/automate/blob/master/e2e/cypress/index.d.ts)

Reply:
I will pull these out in the next PR for adding compliance project update tests